src/test/resources/hello.py

```diff
@@ -0,0 +1 @@
+print("Hello, world!")
```

Comment:
Is this a necessary line to keep?

Reply:
Was deleted in a future commit. Don't read this PR commit-by-commit. Only look at the final changes in the "Files Changed" tab above.